test/url-utils.test.js

```diff
@@ -11,6 +11,31 @@ test('geturl should return url without project if projectname is not provided',
   t.true(result === 'http://unleash-app.com/client/features');
 });
 
+test('geturl should return url with namePrefix if namePrefix is provided', t => {
+  const result = getUrl('http://unleash-app.com', '', 'unleash');
+  t.true(result === 'http://unleash-app.com/client/features?namePrefix=unleash');
+});
+
+test('geturl should return url with namePrefix and project both are provided', t => {
+  const result = getUrl('http://unleash-app.com', 'myProject', 'unleash');
+  t.true(result === 'http://unleash-app.com/client/features?namePrefix=unleash&namePrefix=unleash');
+});
+
+test('geturl should return url with namePrefix, project and tags if all provided', t => {
+  const result = getUrl('http://unleash-app.com', 'myProject', 'unleash', ['tagName:tagValue']);
+  t.true(result === 'http://unleash-app.com/client/features?namePrefix=unleash&namePrefix=unleash&tag=tagName:tagValue');
+});
+
+test('geturl should return url with tags if tags are provided', t => {
+  const result = getUrl('http://unleash-app.com', '', '', ['tagName:tagValue']);
+  t.true(result === 'http://unleash-app.com/client/features?tag=tagName:tagValue');
+});
+
+test('geturl should return url with two tags if two tags are provided', t => {
+  const result = getUrl('http://unleash-app.com', '', '', ['tagName:tagValue', 'tagName2:tagValue2']);
+  t.true(result === 'http://unleash-app.com/client/features?tag=tagName:tagValue&tag=tagName2:tagValue2');
```

Comment:
I think this will work out of the box with express. In other examples we have been using empty square bracket for array bases query params like this:

`http://unleash-app.com/client/features?tag[]=tagName:tagValue&tag[]=tagName2:tagValue2`

Reply:
What do you recommend to do with it?

Reply:
I tried reading up on the standards and it seems to be quite unclear both `brackets` and `repeat` format seems to be supported by express (unleash-server).

I do however think it will be worth while to urlEncode the `tagName:tagValue` params